### ISSUE TITLE
Query strings in output file names are not supported

### ISSUE THREAD

**Issue by nwbauer** [jacobmarshall/webpack-bugsnag-plugin#6]
In my webpack.config, I specify a query string on the output files like so:
```
output: {
  filename: 'js/[name].js?[chunkhash:20]',
  chunkFilename: 'js/[name].chunk.js?[chunkhash:20]',
  path: path.resolve(__dirname, buildDir)
},
```
This naming convention results in the BugsnagSourceMapPlugin to not find any of my sourcemaps.

After glancing at your code, I noticed the reason the plugin couldn't find my sourcemaps was due to the regex used on the `chunk.files` here:
https://github.com/jmshal/webpack-bugsnag-plugin/blob/master/src/BugsnagSourceMapPlugin.js#L40-L41

The ability to support files with query strings can be done by mapping the file names before filtering, so that the query is stripped out.  Something like this:
```
const filesWithoutQuery = chunk.files.map((file) => file.split('?')[0]);
```
and filtering based on those mapped file names. Or updating the regex to account for query strings and then stripping out the query part before upload.

**Comment by jmshal** [comment]
Hey Nicolas,

Sorry about the delayed response. I’ll be honest I’ve never seen the chunk hash used in that way before, so thanks for filing the issue.

Sadly, I’m not going to be able to take a look at this issue until Monday (NZST, Labour Day) - so sorry about that. But PRs are always welcome 😜

**Comment by nwbauer** [comment]
Hi Jacob,

Ok, I can make the PR today.  Fix seems easy enough that I can take a crack at it.

About the output file naming conventions, it is used as a cache-busting strategy. There are basically two camps: (i) put the hash in the filename, and (ii) put the hash as a query string.  While most of the google blog posts give articles explaining (i), the advantage of (ii) is that each deploy simply overwrites files on my server (while webpack does the fetch with the query string so the cloud edge node knows there is a new version).

The plugin is really awesome, so it would be my pleasure to help out!

**Comment by nwbauer** [comment]
Have a branch with commits, but when I try to push I get:

remote: Permission to jmshal/webpack-bugsnag-plugin.git denied to nwbauer

**Comment by jmshal** [comment]
Hey @nwbauer, you'll need to fork the repository first, and then push to your branch to your forked repo. If you need any help, GitHub have some really helpful resources.

- https://help.github.com/articles/fork-a-repo/
- https://help.github.com/articles/creating-a-pull-request-from-a-fork/

If this all seems too much, just let me know, and we can figure out an easier way of getting your code across to this repo 😀

Edit: Oh, and thanks for taking a look! Much appreciated.

**Comment by nwbauer** [comment]
Ah, PR across forks.  Ok cool.

**Comment by jmshal** [comment]
I just released v0.1.1, with your changes within. Thanks again!
